delfin/task_manager/scheduler/schedulers/telemetry/telemetry_job.py

```diff
@@ -77,7 +79,9 @@ def __call__(self, ctx):
                 instance = collection_class.get_instance(ctx, task_id)
                 self.scheduler.add_job(
                     instance, 'interval', seconds=task['interval'],
-                    next_run_time=next_collection_time, id=job_id)
+                    next_run_time=next_collection_time, id=job_id,
+                    misfire_grace_time=int(
+                        CONF.telemetry.performance_collection_interval / 2))
```

Comment:
Any reason we set misfire_grace_time half of interval ?

Reply:
similar question : as misfire_grace_time is grace time to set how much the job’s execution is allowed to be late. Still chances of job miss are there in resource crunch scenario.

Reply:
the reason to set misfire_grace_time half of interval is that I think all the task should be done before next round tasks triggerd, so I think half of intervals is the maximum buffer we can have. But this value of misfire_grace_time could be discussed to make a better value